action_server/src/sema4ai/action_server/_new_project_helpers.py

```diff
@@ -0,0 +1,110 @@
+import io
+import datetime
+import logging
+import os
+import zipfile
+import yaml
+import requests
+from pathlib import Path
+from typing import Dict, List, Union, Optional
+from pydantic.main import BaseModel
+
+from ._settings import get_default_settings_dir
+
+TEMPLATES_METADATA_URL = "https://downloads.robocorp.com/action-templates/action-templates.yaml"
+TEMPLATES_PACKAGE_URL = "https://downloads.robocorp.com/action-templates/action-templates.zip"
+
+ACTION_TEMPLATES_METADATA_FILENAME = "action-templates.yaml"
+
+ACTION_TEMPLATES_DIR = Path(f"{get_default_settings_dir()}/action-templates")
```

Comment:
This shouldn't be a constant (because `get_default_settings_dir()` may create a directory it should be done as lazily as possible).

Reply:
done